### DISCUSSION TITLE
error on console usb app.overlay

### DISCUSSION THREAD

**Discussion by imammashur** [zephyrproject-rtos/zephyr#48508] [Q&A] [answered]
Hi, I have been trying to build an example of **console usb** example and then build it for **thingy91_nrf9160_ns** but got an error

`devicetree error: C:\nrf-workspace\test_thingy91\bridge\hello_world_usb\app.overlay:13 (column 1): parse error: undefined node label 'zephyr_udc0'`

I have been trying this code several weeks ago, but when I try to recreate another example, it seems like this error is something new to me.

Thanks for your help.

**Comment by jfischer-no** [comment, marked as answer]
Only NCS has support for Thingy91 board.
thingy91_nrf9160_ns "board" does not have an USB device controller, but thingy91_nrf52840 does.

**Reply by imammashur** [reply]
Ah, I see, okay, thanks. Now I am able to debug thingy91_nrf9160_ns by uploading bridge connection on thingy91_nrf52840 first. Thanks for your help, really appreciate it.
